config/secrets.yml.template

```diff
@@ -4,15 +4,20 @@
 # The value is the secret string.
 # (Complex objects will be converted to JSON.)
 ---
-foo:
-  base: bar # staging, test, levelbuilder and production
-  production: baz
-qux:
+
+# Example of using `base` alias and setting a different value for production only
+my_key_name:
+  base: my_key_value_shared_by_non_production_systems # staging, test, levelbuilder and production (note: does not include development)
```

Comment:
should we take production out of this list, since this is value is for non-production systems?

Reply:
ah yes that makes sense! Especially since it's stated in the initial block of the template which environments it covers. I'll update!

Reply:
updated here: https://github.com/code-dot-org/code-dot-org/pull/42692/commits/92c03672c57033615fa1acb6a35382c4fb3044af